static/js/lib/validation.js

```diff
@@ -44,17 +44,30 @@ export const changePasswordFormValidation = yup.object().shape({
       user_email === email ? schema.required() : schema.notRequired()
     ),
 
-  newPassword: newPasswordFieldValidation
+  newPassword: yup
+    .string()
     .label("New Password")
     .when(["email", "user_email"], (email, user_email, schema) =>
-      user_email === email ? schema.required() : schema.notRequired()
+      user_email === email
+        ? schema
+          .matches(/^(?=.*[0-9])(?=.*[a-zA-Z]).*$/, {
+            message: "Password must contain at least one letter and number"
+          })
+          .required()
+          .min(8)
+        : schema
+          .matches(/^.*$/)
+          .notRequired()
+          .min(0)
```

Comment:
Are you sure about the `min(0)`? I can understand the `required/notRequired` change though `min` should not differ. Correct me if I'm wrong, please.

Reply:
yes, I have tested that its working fine without changing `min(0)` email.